test/uk/gov/hmrc/gform/graph/FormTemplateBuilder.scala

```diff
@@ -17,10 +17,14 @@
 package uk.gov.hmrc.gform.graph
 
 import cats.data.NonEmptyList
+import uk.gov.hmrc.gform.sharedmodel.{ LangADT, LocalisedString }
 import uk.gov.hmrc.gform.sharedmodel.formtemplate._
 import uk.gov.hmrc.gform.sharedmodel.formtemplate.destinations.Destinations.DmsSubmission
 
 object FormTemplateBuilder {
+  private def toLocalisedString(string: String) =
```

Comment:
There are 17 implementation of `toLocalisedString`. Let's write only one (and use lifting to get other variants of the same function. ie. the one which works with `Option`, `List` etc.)